crates/sui-graphql-rpc/src/types/address.rs

```diff
@@ -144,6 +144,7 @@ impl Address {
         before: Option<transaction_block::Cursor>,
         relation: Option<AddressTransactionBlockRelationship>,
         filter: Option<TransactionBlockFilter>,
+        within_checkpoints: Option<u64>,
```

Comment:
I'm interpreting this as the scan limit parameter -- if so, that's measured in units of transactions, rather than checkpoints, so we would need to change the name here -- is there strong opposition to `scan_limit` being the name? I figured that it's useful to have a standard concept for this without trying to mention that it's over transactions/checkpoints/events etc so that when people see it on the events queries they know what it has the same meaning as it did here.

Reply:
Funnily enough this was the initial implementation. Then I saw your discussion with Hayes, and then I changed the implementation to checkpoints instead of txs :facepalm;